lerobot/configs/robot/stretch.yaml

```diff
@@ -0,0 +1,25 @@
+_target_: lerobot.common.robot_devices.robots.stretch.StretchRobot
+robot_type: stretch3
+
+cameras:
+  # TODO(aliberts): hardware currently not working, uncomment and test this when fixed
```

Comment:
Did this get resolved? This is specific to your robot, right?

Reply:
Replacement board is on the way ;)

Reply:
Glad to hear it!